docs/releng.md

```diff
@@ -13,4 +13,10 @@ This document is for Release Engineering, on how to maintain this repo, and how
    ```
 4. Create a pull request, adding a new signing manifest to the [signing manifest directory](https://github.com/mozilla-releng/adhoc-signing/tree/master/signing-manifests). Use the [template](https://github.com/mozilla-releng/adhoc-signing/blob/master/signing-manifests/example.yml.tmpl) and create a new `.yml` file.
 5. Get review, and merge.
-6. Promote your manifest to get a valid release signature. This will require some shipit changes that aren't yet implemented.
+6. Promote your manifest to get a valid release signature. Someday these steps will be in shipit, but for now:
+   - Go to the merge taskgroup. Generally this is linked from the status symbol of the latest merge to master: a yellow dot for in-progress, a red X for failure, and a ![green checkmark](screenshot1.png) for success. Click it.
+   - Click on `details` of the decision task, then `View task in taskcluster` to go to the decision task. This task will need to be green before we can proceed.
+   - Go to the task group view: Either click on `Task Group` at the top left, or change the `tasks/TASKID` to `tasks/groups/TASKID` in the url bar.
+   - Click the vertical three dots in the lower right, and choose `Promote an adhoc signature`. You will need to be signed in for this to work. The sign-in link is in the top right.
```

Comment:
I don't see that option on the ... menu, is there a set of people who have this enabled ?

Reply:
Errr, PEBKAC. Helps if you look at the group for the merge commit, rather than the PR commit.